src/System.Net.Security/src/System/Net/NegotiationInfoClass.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    // This class is used to determine if NTLM or
```

Comment:
See other comment: I don't think we should create a partial class here. (Maybe a different class to hold the strings?)